### PR TITLE
[LIB-509][ci skip] socialLogin method for Account object

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: src/account.js [hunk 1]
```diff
@@ -68,6 +69,32 @@ const Account = stampit().compose(Request)
...
+    * @returns {Promise}
+
+    */
+    socialLogin(payload = {}, setAccountKey = true) {
+      const config = this.getConfig();
+      const path = this._account.socialLoginPath(payload.instanceName, payload.backend);
+
+      return this.makeRequest('POST', path, {payload}).then((user) => {
+        if (setAccountKey === true) {
+          config.setAccountKey(user.account_key);
```
Comment (MariuszWisniewski) [inline src/account.js]: Why is it called account key? Not `user_key`?

Comment (jakubbilko) [inline src/account.js]: @lifcio my bad, fixed in 1b2b83caf01a182262cfc27480608f9e8a24e8d4

File: src/account.js [hunk 2]
```diff
@@ -68,6 +69,32 @@ const Account = stampit().compose(Request)
...
+
+    * @param {Object} payload
+    * @param {String} payload.instanceName
+    * @param {String} payload.backend
+    * @param {String} payload.access_token
+    * @param {Boolean} [setUserKey = true]
+    * @returns {Promise}
+
+    */
+    socialLogin(payload = {}, setAccountKey = true) {
```
Comment (MariuszWisniewski) [inline src/account.js]: One left here - setAccountKey -> user key

File: src/account.js [hunk 3]
```diff
@@ -68,6 +69,32 @@ const Account = stampit().compose(Request)
...
+      const config = this.getConfig();
+      const path = this._account.socialLoginPath(payload.instanceName, payload.backend);
+
+      return this.makeRequest('POST', path, {payload}).then((user) => {
+        if (setAccountKey === true) {
+          config.setAccountKey(user.user_key);
```
Comment (MariuszWisniewski) [inline src/account.js]: And here setAccountKey -> user key

**Comment by MariuszWisniewski** [comment]
No further comments from me ;)

**Comment by jakubbilko** [comment]
:smile:

**Pull request merged**